frontend/src/components/pages/worker/WorkerCarsPage.jsx

```diff
@@ -1,7 +1,53 @@
+import { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
 export function WorkerCarsPage() {
+    const navigate = useNavigate()
+    const [vehicles, setVehicles] = useState(null)
+    useEffect(() => {
+        get_vehicles()
+    }, [])
+
+    function get_vehicles() {
+        fetch('/api/vehicles/', {
+            method: 'GET',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+        })
+            .then(response => response.json())
+            .then(data => {
+                data.sort((a, b) => a.vehicle_id - b.vehicle_id);
+                setVehicles(data)
+            })
+            .catch(error => {
+                console.log('Error getting vehicle info', error)
+            })
+    }
+
     return (
-        <div className='worker_cars_page page_content'>
-            <h2>Cars</h2>
-        </div>
+        <div className='worker_vehicles_page page_content'>
+            <h2>Vehicles</h2>
+            <table>
```

Comment:
Zamiast `<table>` użyj naszego komponentu `<Table>`

Reply:
No fajnie by było

Reply:
To jednak zapomnij, że o to pytałem Xd

Reply:
ale te moje jest